lib/main.dart

```diff
@@ -26,6 +28,7 @@ Future _initSdks() async {
   WidgetsFlutterBinding.ensureInitialized();
   await Logger.init();
   await Config.initialize();
+  Hive.init(Config.appDirectoryPath);
```

Comment:
It's a provider, I'd initialize it in the provider module.

Reply:
It's not a provider it initializes a singleton. The return type is a void and not a Hive instance